src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyOptions.php

```diff
@@ -67,16 +67,12 @@ public function setTitle(string $title): self
 
     public function setStringPriority(string $priority): self
     {
-        switch ($priority) {
-            case Notification::IMPORTANCE_URGENT:
-                return $this->setPriority(self::PRIORITY_URGENT);
-            case Notification::IMPORTANCE_HIGH:
-                return $this->setPriority(self::PRIORITY_HIGH);
-            case Notification::IMPORTANCE_LOW:
-                return $this->setPriority(self::PRIORITY_LOW);
-            default:
-                return $this->setPriority(self::PRIORITY_DEFAULT);
-        }
+        return match ($priority) {
```

Comment:
Just in case, `setPriority` allows for one more priority, which is not handled here: `PRIORITY_MIN`

![image](https://github.com/symfony/symfony/assets/870747/63bc8418-af96-46fb-be87-f1e79c587e51)

Reply:
In that case, the `default` value is applied to it, so it's fine.